org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/javaeditor/codemining/JavaMethodParameterCodeMining.java

```diff
@@ -40,6 +41,17 @@ public JavaMethodParameterCodeMining(Expression parameterNode, int parameterInde
 		}
 	}
 
+	public JavaMethodParameterCodeMining(Expression parameterNode, int parameterIndex, IMethodBinding mbinding, boolean isVarargs, ICodeMiningProvider provider) {
```

Comment:
Again, I think we have a method binding in ever call site of this constructor. Why bother with the `IMethod` at all?